app/views/exam_templates/_generate_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_for [@assignment, exam_template], url: generate_assignment_exam_template_path(id: exam_template.id),
+             html: { multipart: true } do |f| %>
```

Comment:
You shouldn't need the `multipart` here because there isn't file data being submitted.